centos-ci/bootstrap.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+if [[ $(id -u) != 0 ]]; then
+   echo "$0 needs to be run as root" ; exit 1
+fi
+
+pip install ansible==2.2.0
```

Comment:
Hmm, I may have been too hasty on the merge here: while this line will succeed on Fedora, isn't it going to fail on CentOS 7, since there's no "pip" in the default image?

We'll likely need to bootstrap `pip` itself first via `curl "https://bootstrap.pypa.io/get-pip.py" | python` as covered by Option #2 in https://www.liquidweb.com/kb/how-to-install-pip-on-centos-7/